### ISSUE TITLE
Not c++11 compatible

### ISSUE THREAD

**Issue by oscarthorn** [AcademySoftwareFoundation/openexr#1930]
Hi!

Just wanted to mention that the docs: https://openexr.com/en/latest/install.html#install say that a compiler that supports c++11 is needed but 693a7e4 introduces `#include <filesystem>` which is a c++17 header. So either support for c++11 and c++14 was dropped but then the docs should change or this was an oversight.

Cheers!

**Comment by dylan-reedy** [comment]
Bump for visibility. This also caught me by surprise!

**Comment by peterhillman** [comment]
Indeed, it seems there's a mismatch between the c++ versions mentioned in the docs, the c++ versions the CI system tests with, and the minimum c++ version the code actually requires.

@oscarthorn and @dylan-reedy do you require c++11 or c++14 compatibility, or did you just spot the discrepancy?

**Comment by oscarthorn** [comment]
@peterhillman I don't need it, just noticed it!

**Comment by meshula** [comment]
Tagged with documentation to note that we should update docs.

**Comment by dylan-reedy** [comment]
@peterhillman I also just happened to notice when trying to build on an older system, no specific need.

**Comment by ryandesign** [comment]
> the docs: https://openexr.com/en/latest/install.html#install say that a compiler that supports c++11 is needed

The install.rst file in the website branch still says "C++11":

https://github.com/AcademySoftwareFoundation/openexr/blob/c6aec020574a77925bd4adf4175637997f904ffd/docs/install.rst?plain=1#L56

However the web site now says "C++17". So... the web site is not built from the website branch?

**Comment by peterhillman** [comment]
The website is built from the website folder of the main branch. The install page on openexr.com is built from this file:
https://github.com/AcademySoftwareFoundation/openexr/blob/main/website/install.rst

The website branch is a stale branch used during a website redesign. @cary-ilm perhaps the branch should be deleted or renamed now to avoid confusion?

**Comment by cary-ilm** [comment]
Following up on old/stale issues. 

I think this has been resolved.  The website now references C++17, and there is no "website" branch, so it must have gotten deleted.
